tests/test_transforms/test_math/test_lambda_transform.py

```diff
@@ -75,15 +77,15 @@ def test_save_transform(ts_non_negative, transform_original, transform_function,
 def test_nesessary_inverse_transform(ts_non_negative):
     with pytest.raises(ValueError, match="inverse_transform_func must be defined, when inplace=True"):
         transform = LambdaTransform(in_column="target", inplace=True, transform_func=lambda x: x)
-        ts_non_negative.fit_transform([transform])
+        ts_non_negative = transform.fit_transform(ts_non_negative)
```

Comment:
Omit the left side, to emphasize that transforms work inplace(fix it everywhere)